config/application.rb

```diff
@@ -100,6 +100,17 @@ class Foreman::Consoletie < Rails::Railtie
 
 module Foreman
   class Application < Rails::Application
+    config.load_defaults 6.1
```

Comment:
While they call it with `target_version.to_s`, I still think it's better to use strings. Version numbers are weird and not really floats.
```suggestion
    config.load_defaults '6.1'
```